File: Series_0/series0_code/midpoint/plot_error.m
errors = load('series0_1_d_errors.txt');
numberOfIntervals = load('series0_1_d_numbers.txt');

loglog(numberOfIntervals, errors, '*-');
title('Error plot for ex. 0.1d)')
xlabel('n')
ylabel('|I(f)-I_n(f)|')

grid on
grid minor

File: Series_0/series0_code/midpoint/plot_error.py
import numpy
import pylab
errors = numpy.loadtxt('series0_1_d_errors.txt')
numberOfIntervals = numpy.loadtxt('series0_1_d_numbers.txt')

pylab.loglog(numberOfIntervals, errors, '-*')
pylab.title('Error plot for ex. 0.1d)')
pylab.xlabel('n')
pylab.ylabel('|I(f)-I_n(f)|')
pylab.grid('on')
pylab.show()
